tests/features/contact_form.feature

```diff
@@ -12,8 +12,6 @@ Feature: Contact Form
       | contact_form |
     When I am on "contact"
     Then I should see "Contact - European Commission" in the "title" tag
-    When I fill in "Your name" with "Chuck"
```

Comment:
I agree with @gillesdeudon this two lines need to be present because we need to make sure that anonymous users can add their name and email